SUMMARY.md

```diff
@@ -15,7 +15,6 @@
 
 ## Writing a Driver
 
-* [Creating a Driver](driver/kickoff.md)
```

Comment:
May be for the visibility we could keep this section here, but point it to our new [./driver/sdk.md@bootstrapping-a-new-driver](https://github.com/bblfsh/documentation/pull/14/files#diff-7df754cc1aa08d4f35abb709f2592434R25) section?